src/clusterfuzz/_internal/google_cloud_utils/storage.py

```diff
@@ -1377,17 +1378,24 @@ def get_arbitrary_signed_upload_url(remote_directory):
       get_arbitrary_signed_upload_urls(remote_directory, num_uploads=1))[0]
 
 
-def maybe_parallel_map(func, arguments):
+def parallel_map(func, argument_list):
```

Comment:
The if statement is being removed because the rearch is complete, right?

Reply:
Correct!